src/pkg/manifests/package:pkg:depot.p5m

```diff
@@ -61,7 +61,6 @@ file path=var/log/pkg/depot/access_log owner=pkg5srv mode=0644 preserve=true
 file path=var/log/pkg/depot/error_log owner=pkg5srv mode=0644 preserve=true
 file path=var/log/pkg/depot/rewrite.log owner=pkg5srv mode=0644 preserve=true
 license cr_Oracle license=cr_Oracle
-depend type=require fmri=package/pkg@$(PKGVERS)
```

Comment:
We should add one dependency on IPS main package from depot package though. Or not?

Reply:
```
@@ -33,20 +33,19 @@
...
-depend fmri=package/pkg@0.5.11,5.11-0.151.1.8.1 type=require
...
 depend fmri=pkg:/package/pkg@0.5.11-0.151.1.8.1 type=require
```